src/AutoBuild.jl

```diff
@@ -1522,8 +1500,9 @@ function build_project_dict(name, version, dependencies::Array{Dependency})
     end
     # Always add Libdl and Pkg as dependencies
```

Comment:
Still mentions Pkg

Reply:
Yep, we're still going to express a dependency on `Pkg` because if it's an older version of Julia running, it will need to import `Pkg.Artifacts`, not `Artifacts`.